plugins/calendar/src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/calendar/CalendarLogic.java

```diff
@@ -131,26 +132,19 @@ public String getEventsToday() {
 		List<String> eventList = new ArrayList<>();
 		try {
 			LocalDateTime now = LocalDateTime.now();
```

Comment:
use the service `Environment` to get the current localdatetime.
Pro:
- you can mock Environment and test the code with any localdatetime you want.